example/node/index.js

```diff
@@ -56,9 +56,9 @@ async function run() {
     );
     return;
   }
-  if (!process.env.S1GRD_LAYER_ID) {
+  if (!process.env.S1GRDIW_LAYER_ID) {
     printOut(
-      'Example needs id of Sentinel-1 GRD layer to run. Please set env var S1GRD_LAYER_ID and run again',
+      'Example needs id of Sentinel-1 GRD layer to run. Please set env var S1GRDIW_LAYER_ID and run again',
```

Comment:
We kinda missed it before, but the Acquisition mode must be set in the configuration, so I suggest we change this to: `Example needs id of Sentinel-1 GRD layer with Acquisition mode set to IW to be able to run.`

Reply:
True, but I would suggest we postopne these fixed until after we have more of the datasets ready, because we will need to com up with a consistent way of dealing with more than just S-2 and S-1, if you agree?

Reply:
I agree, resolving.